Hazel/src/Hazel/Core.h

```diff
@@ -36,8 +36,17 @@ namespace Hazel {
 
 	template<typename T>
 	using Scope = std::unique_ptr<T>;
+	template<typename T, typename... Args>
+	constexpr inline Scope<T> CreateScope(Args&&... args)
```

Comment:
constexpr implies inline, so potentially redundant having both

Reply:
You are correct! The inline can be removed for both.